configs/components/recom/recom.yaml

```diff
@@ -167,9 +187,12 @@ config_files:
         recom:  recom
 
 config_sources:
-        recom: "${namelist_dir}/namelist.recom"
-#        io.recom.ciso: "${namelist_dir}/namelist.io.recom.ciso"
+        recom:  "${namelist_dir}/${recom_namelist}${namelist_sce}"
 
+config_in_work:
+        recom: "namelist.recom"
+
+#4.7.22 Ying: why deleting 'restart' in filename?
```

Comment:
```suggestion
```
It's a default (first level of the yaml), so that when you have a default restart you use `fesom.<YEAR>.bio.nc`. This is for example the case of @chrisdane, where his file in the work directory in AWI-CM1-REcoM needs to be called that way. However, when you run AWIESM you set, in the recom section of `awiesm.yaml` `restart_name: .restart`:
https://github.com/esm-tools/esm_tools/blob/0a0e010e77dcf7858153af26132e862fa3653c14/configs/setups/awiesm/awiesm.yaml#L360